### ISSUE TITLE
Backup over HTTP API

### ISSUE THREAD

**Issue by deinok** [influxdata/influxdb#22400]
I was looking to execute a backup order over the HTTP API.
Not sure if this is already implemented or not. I will explain my case.

We have an InfluxDB2 inside a Docker Container. We can't relay on Internet as it runs physicaly in the middle of nowhere. But most of the time the host and the container has internet.

We have a Docker Volume where we save the backups. But we would like to force a backup without using SSH, etc... just via the common HTTP API (With the correct permisions token)

**Comment by wbaker85** [comment]
I don't think you'll be able to do a backup without using the CLI since the CLI does some special handling of the received files to process them and make subsequent requests.

Any reason you couldn't run the CLI on the same local network to do the backup? Or are you looking for an HTTP endpoint that would trigger the server to do the backup entirely server-side and save it somewhere?

**Comment by deinok** [comment]
Yes, I am looking to some kind of HTTP endpoint to trigger that backup.

My general idea is that the docker volume with the backups will be shared with our App. So it can move it to a remote storage service.

**Comment by wbaker85** [comment]
Thanks for the clarification. Since we don't currently have this ability, I've tagged this as a feature request.

**Comment by aleksrosz** [comment]
IMHO you can use Kapacitor to copy data from one database to another. Maybe that is solution that you need?

**Comment by deinok** [comment]
@aleksrosz It doesn't fix our issue Kapacitor
